std_msgs/Header header

int32 id
float32 nx
float32 ny
float32 nz
float32 d 
geometry_msgs/Vector3 plane_orientation
geometry_msgs/Vector3[] plane_points